pkg/build/generated/clientset/typed/build/v1/buildconfig_expansion.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+import (
+	"errors"
+	"net/url"
+
+	build "github.com/openshift/origin/pkg/build/apis/build/v1"
+)
+
+type BuildConfigExpansion interface {
```

Comment:
ignore this, this is going away

Reply:
> ignore this, this is going away

going away how?  I'd rather not have non-generated code in my generated package and since this can sit on top and if another project *really* wanted it, they could vendor a separate "client helpers" package, I'd rather keep it out of exapnsions.

Reply:
yeah, I'm moving this to pkg/build/client/internalversion for now.